blog/20240814-simplex-chat-vision-funding-v6-private-routing-new-user-experience.md

```diff
@@ -0,0 +1,183 @@
+- Block messages of a member for yourself only.
+- Block members for all other members - only admins and group owners can do that.
+
+With this version you can now select multiple messages at once and delete or moderate them, depending on your role in the community. The current version limits the number of messages that can be deleted to 20 - this limit will be increased to 200 messages in the next version.
+
+Also, this version makes profile images of the blocked members blurred, to prevent the abuse via inappropriate profile images.
+
+#### Increase font size
+
+Android and desktop apps now allow to increast font size inside the app, without changing the system settings. Desktop in addition to that allows to zoom the whole screen - it can be helpful on some systems with a limited support of high density displays.
+
+These settings can be changed via appearance settings.
+
+### New media options
+
+#### Play from the chat list
+
+Now you can interact with the media in the last message directly from the list of the chats.
+
+This is very convenient – when somebody sends you a voice message or a video, they can be played directly from the list of chats, without opening a conversation. Similarly, an image can be opened, a file can be saved, and the link with preview can be opened in the browser.
+
+And, in some circumstances, this is also more private, as you can interact with the media, without opening the whole conversation.
+
+We will add the option to return missed calls from the chat list in the next version.
+
+#### Blur for better privacy
+
+You can set all images and videos to blur in your app, and unblur them on tap or on hover in desktop app. The blur level can be set in Privacy and security settings.
+
+#### Share from other apps
+
+Not much to brag about, as most iOS apps have it, and users expect it to be possible.
+
+But iOS makes it much harder to develop the capability to share into the app than Android, so it's only in this version you can share images, videos, files and links into SimpleX Chat from other apps.
+
+### Improved networking and reduced battery usage
+
+This version includes the statistics of how your app communicates with all servers when sending and receipving messages and files. This information also includes the status of connection to all servers from which you receive messages - whether the connection is authorized to push messages from server to your device, and the share of these active connections.
+
+Please note, that when you send a message to a group, your app has to send it to each member separately, so sent message statistics accounts for that - it may seem to be quite a large number if you actively participate in some large groups. Also, message counts not only include visible messages you receive and send, but also any service messages, reactions, message updates, message deletions, etc. - this is the correct reflection of how much traffic your app uses.
```

Comment:
```suggestion
Please note, that when you send a message to a group, your app has to send it to each member separately, so sent message statistics account for that - it may seem to be quite a large number if you actively participate in some large groups. Also, message counts not only include visible messages you receive and send, but also any service messages, reactions, message updates, message deletions, etc. - this is the correct reflection of how much traffic your app uses.
```